src/test/java/com/ericsson/ei/flowtests/FlowTestBase.java

```diff
@@ -77,25 +82,33 @@
     private static ObjectMapper objectMapper = new ObjectMapper();
 
     private static HashMap<String, FlowTestConfigs> configsMap = new HashMap<String, FlowTestConfigs>();
+    
+    protected static boolean systemTest = false;
```

Comment:
Can't this be changed to property.
     protected static boolean systemTest = false; 
    @Value("${ei.backendInstancesFilePath:#{null}}")
    private Boolean systemTestProperty;

If (systemTestProperty) {
    systemTest = true;
} 

Then its default systemTest false and if systemTestProperty is provided, then its true.
Example: java -jar <appl name>.jar -systemTestProperty=true

Or something similar. Pick better property name, if you change to this solution.

Reply:
Done.